src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -887,23 +885,24 @@ public virtual Task<bool> TryUpdateModelAsync<TModel>([NotNull] TModel model)
             [NotNull] params Expression<Func<TModel, object>>[] includeExpressions)
            where TModel : class
         {
-            if (BindingContextProvider == null)
+            if (ActionContext.BindingContext == null)
```

Comment:
wondering if these provide any value ```TryUpdateModel``` would be called from within an action by which time this would have been initialized.